src/main/data/facilities.json

```
@@ -863,7 +863,12 @@
   },
   {
     "code": "CJP",
-    "description": "CJ POWERCAST ART AND TECHNOLOGY"
+    "description": "CJ POWERCAST ART AND TECHNOLOGY",
+    "obsolete": true,
+    "obsoletedBy": [
+      "CJO"
+    ],
```

```suggestion
    ]
```

@japierce1028 Extra comma